accessibility/keyboard-accessibility/keyboard-accessibility.md

```diff
@@ -0,0 +1,56 @@
+# Keyboard Accessibility
+
+Sparkbox recognizes the importance of creating a web that is accessible to everyone, so we prioritize developing and testing for accessibility as an integral part of our process. In addition to automated testing, our developer conduct ongoing accessibility tests as they build their projects.  Manually testing content and custom widgets for keyboard only accessibility during this phase allows issues to be identified and remediated early, before they become a problem for a user.
```

Comment:
I’m not sure what a “custom widget” is or referring to. Would it be better to say site features?

Reply:
This is the terminology DeQue uses for a something like a custom form element that doesn't use native HTML elements.  But I agree, it's not clear.  I'll try to come up with a way to make it more obvious what I am talking about.